.github/workflows/ci.yml

```diff
@@ -9,33 +9,76 @@ jobs:
   ci:
     name: CI
     runs-on: ubuntu-latest
-    needs: [ci-linux]
-    steps: 
+    needs: [check, ci-linux, ci-serde]
+    steps:
       - name: Done
         run: exit 0
+
+  check:
+    name: Cargo check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        TARGET: [x86_64-unknown-linux-gnu, x86_64-apple-darwin, x86_64-pc-windows-msvc]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+          target: ${{ matrix.TARGET }}
+
+      - name: Cache Dependencies
+        uses: Swatinem/rust-cache@v1
+        with:
+          key: ${{ matrix.TARGET }}
```

Comment:
should add a `sharedKey: prebuilt` here, and in the prebuild job the `key` should be `prebuilt`

Reply:
No, Prebuilt and Check tasks are independent and must have different caches.